common/views/components/MediaObjectList/MediaObjectList.tsx

```diff
@@ -1,14 +1,18 @@
 import { MediaObjectType } from '../../../model/media-object';
 import MediaObject from '../../components/MediaObject/MediaObject';
 
-const MediaObjectList = ({ items }: { items: Array<MediaObjectType> }) => {
+const MediaObjectList = ({
+  items,
+}: {
+  items: Array<MediaObjectType>;
```

Comment:
```suggestion
  items: MediaObjectType[];
```
👇 ?
https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/array-type.md